tests/linux/test_snap_utils.cpp

```diff
@@ -34,14 +35,14 @@ TEST(SnapUtils, test_is_confined_when_snap_dir_set)
 {
     mpt::SetEnvScope env("SNAP", "/tmp");
```

Comment:
```suggestion
    mpt::SetEnvScope env("SNAP", "/tmp");
    mpt::SetEnvScope env("SNAP_COMMON", "/tmp");
```

Reply:
Although I know it's harmless, I'm not sure why we need to set `SNAP_COMMON` for this test.  Could you please explain why you suggested this?

Reply:
Sorry, this was actually an oversight on my part. I noticed there was no analogous test checking `mu::snap_common_dir` didn't throw and it seemed that adding that var here would do it, but it won't because that function is not being called. So my suggestion would instead be to have that analogous test.

Reply:
Right, I can either add the second call _with_ your suggestion applied or just add a second test.

Reply:
Yes, exactly.

Reply:
Done.  I made another test for this.